src/AssemblerAlign3.cpp

```diff
@@ -95,7 +95,14 @@ void Assembler::alignOrientedReads3(
 
     }
 
-
+    if ((downsampledSequences[0].data_begin == downsampledSequences[0].data_end) or
```

Comment:
I would avoid using these undocumented SeqAn fields. Every time we add something to `downsampledSequences` we also add it to `downsampledMarkers`, so I would replace this with 

```
if(downsampledMarkers[0].empty() or downsampledMarkers[1].empty())
```